### PR TITLE
Updated the link to point to correct resource

### PR DESCRIPTION
**Pull request by joinimran**
Updated the link to point to the correct resource. 

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1850

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

@mypil FYI Please.

**Comment by mypil** [comment]
@JohanFreelancer9 - please do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/offering-ISO-27001.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ ISO/IEC 27001 is a security standard that formally specifies an Information Secu
 
 The international acceptance and applicability of ISO/IEC 27001 is the key reason why certification to this standard is at the forefront of Microsoft’s approach to implementing and managing information security. Microsoft’s achievement of ISO/IEC 27001 certification points up its commitment to making good on customer promises from a business, security compliance standpoint. Currently, both Azure Public and Azure Germany are audited once a year for ISO/IEC 27001 compliance by a third-party accredited certification body, providing independent validation that security controls are in place and operating effectively.
 
-Learn about the benefits of ISO-Iec-27001 on the Microsoft Cloud: [Download the ISO/IEC 27001:2013](https://go.microsoft.com/fwlink/p/?linkid=2100068)
+Learn about the benefits of ISO-Iec-27001 on the Microsoft Cloud: [Download the ISO/IEC 27001:2013](https://servicetrust.microsoft.com/ViewPage/MSComplianceGuide?command=Download&downloadType=Document&downloadId=47d89200-b24b-491d-b657-7c523ddfb6f9&docTab=4ce99610-c9c0-11e7-8c2c-f908a777fa4d_ISO_Reports)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/offering-ISO-27001.md]:
```suggestion
Learn about the benefits of ISO/IEC 27001 on the Microsoft Cloud: [Download the ISO/IEC 27001:2013](https://servicetrust.microsoft.com/ViewPage/MSComplianceGuide?command=Download&downloadType=Document&downloadId=47d89200-b24b-491d-b657-7c523ddfb6f9&docTab=4ce99610-c9c0-11e7-8c2c-f908a777fa4d_ISO_Reports)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
@mypil - Ready to go live

**Comment by mypil** [comment]
@robmazz - PR has been copy edited and is ready for final review. This is based on issue #1850.

Thank you.

cc: @kenwith

**Pull request merged**